frontend/public/components/utils/resource-log.jsx

```diff
@@ -94,7 +111,16 @@ export class ResourceLog extends React.Component {
     return null;
   }
 
+  fetchPodLogLinks() {
+    coFetchJSON('api/kubernetes/apis/console.openshift.io/v1/consolepodloglinks')
+      .then(podLogLinks => this.setState({podLogLinks: podLogLinks.items}))
+      .catch(e => this.setState({error: e}));
```

Comment:
If the CRD does not exist, an alert is rendered.  Maybe we don't want to catch errors here?

![Screen Shot 2019-06-20 at 10 59 25 AM](https://user-images.githubusercontent.com/895728/59874719-f6ed2b00-936c-11e9-8ea4-c0e12dfc4a4a.png)

Reply:
At least should let a 404 pass. Thinking about other errors, if they would matter.

Reply:
We should set a flag during discovery and only try to get pod log links if the resource exists.

Reply:
https://github.com/spadgett/console/blob/a69288f379eb9817b2fc798c977ad28d17a6a3a5/frontend/public/reducers/features.ts#L26-L35

Reply:
Thanks, @spadgett!  Feature detection added.